lib/auth/auth.go

```diff
@@ -208,10 +208,11 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 			WindowsDesktops:       cfg.WindowsDesktops,
 			SessionTrackerService: cfg.SessionTrackerService,
 		},
-		keyStore:     keyStore,
-		getClaimsFun: getClaims,
-		inventory:    inventory.NewController(cfg.Presence),
-		AppPublisher: cfg.AppPublisher,
+		keyStore:               keyStore,
+		getClaimsFun:           getClaims,
+		inventory:              inventory.NewController(cfg.Presence),
+		AppPublisher:           cfg.AppPublisher,
+		RemoteAccessAppWatcher: publisher.NewRemoteAccessAppWatcher(context.Background(), cfg.Events, cfg.AppPublisher),
```

Comment:
I would call it IdemeumRemoteResourceWatcher

Reply:
will change it